assets/src/activation-notice/app/components/image.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../config';
+
+const Img = styled.img`
+  transform: rotate(${(props) => props.$rotationAngle});
+  margin-top: 60px;
+`;
+
+function Image({ name, name2x, ...props }) {
+  const { cdnURL, isRTL } = useConfig();
+  return (
+    <Img
+      src={`${cdnURL}images/plugin-activation/${name}`}
+      srcSet={`${cdnURL}images/plugin-activation/${name}, ${cdnURL}images/plugin-activation/${name2x} 2x`}
+      alt=""
+      $rotationAngle={isRTL ? '-30deg' : '30deg'}
```

Comment:
?

Reply:
What‘s unclear @spacedmonkey?

Props prefixed with a dollar sign are so-called transient props, a styled-components feature.

Reply:
See https://styled-components.com/docs/api#transient-props

Reply:
👍 When a while since I looked at styled components. The more you know.